Elias/exercise-4.js

```diff
@@ -0,0 +1,42 @@
+//Variables
+let apples = 20;
+let oranges = 30;
+let mangos = 5;
+
+// Are apples and oranges equal?
+console.log(apples === oranges); //==> false
+
+//Are apples and oranges unequal?
+console.log(apples !== oranges); //==> true
+
+// are apples greater than oragnes?
+console.log(apples > oranges); //==> false
+
+// are apples less than oranges?
+console.log(apples < oranges); //==> true
+
+// are oranges greater than apples?
+console.log(oranges > apples); //==> true
+
+// is mangos added to oranges greater than mangoes multiplied by apples?
+let sumApMa = apples * mangos;
+let sumMaOr = mangos + oranges;
+console.log(sumApMa > sumMaOr); //==> true
+
+// is apples subtracted by mangos less than oranges divided by mangos?
+console.log(apples - mangos < oranges / mangos); //==> false
+
+// are oranges equal to mangos and apples?
+console.log(mangos === apples && mangos === oranges && apples === oranges); //==> false
```

Comment:
In this case you dont need to use && in console.log and the simple answer is 
console.log(apples == oranges == mangoes);

Reply:
Ok got it, thanks.